index.js

```diff
@@ -112,8 +112,18 @@ FirebaseServer.prototype = {
 		function authData() {
 			var data;
 			if (authToken) {
+				var decodedToken;
 				try {
-					data = server._tokenValidator.decode(authToken).d;
+					decodedToken = server._tokenValidator.decode(authToken);
+					if ('d' in decodedToken) {
+						data = decodedToken.d;
+					} else {
+						data = {
+							uid: decodedToken.sub,
```

Comment:
You are right.
Both have the same value, but `sub` is a standard JWT claim, while `user_id` is firebase-specific.
What if I use `sub` as a fallback for `user_id`?